modules/compute/gke-node-pool/gpu-direct-workload/scripts/enable-tcpx-in-workload.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import yaml
+import argparse
+
+def main():
+    parser = argparse.ArgumentParser(description="TCPX Job Manifest Generator")
+    parser.add_argument("-f", "--file", required=True, help="Path to your job template YAML file")
+    parser.add_argument("-r", "--rxdm", required=True, help="RxDM version")
+
+    args = parser.parse_args()
+
+    # Get the YAML file from the user
+    if not args.file:
+        args.file = input("Please provide the path to your job template YAML file: ")
+
+    # Get component versions from user
+    if not args.rxdm:
+        args.rxdm = input("Enter the RxDM version: ")
+
+    # Load and modify the YAML
+    with open(args.file, "r") as file:
+        job_manifest = yaml.load(file, Loader=yaml.BaseLoader)
+
+    # Update annotations
+    add_annotations(job_manifest)
+
+    # Update volumes
+    add_volumes(job_manifest)
+
+    # Update tolerations
+    add_tolerations(job_manifest)
+
+    # Add tcpx-daemon container
+    add_tcpx_daemon_container(job_manifest, args.rxdm)
+
+    # Update environment variables and volumeMounts for GPU containers
+    update_gpu_containers(job_manifest)
+
+    # Generate the new YAML file
+    updated_job = str(yaml.dump(job_manifest, default_flow_style=False, width=1000, default_style="|", sort_keys=False)).replace("|-", "")
+
+    new_file_name = args.file.replace(".yaml", "-tcpx.yaml")
+    with open(new_file_name, "w", encoding="utf-8") as file:
+        file.write(updated_job)
+
+    # Step 7: Provide instructions to the user
+    print("\nA new manifest had been generated and updated to have TCPX enabled based on the provided worklad,")
+    print("It can be found in the same path as the original workload file with name ending \"-tcpx\"")
```

Comment:
Can we print out the exact path of the file ? I see that we have added the command below on how to create workload, but we perhaps can print out from this command as well so that the customer can quickly get a runnable command as an output from this script .

Reply:
done